CMake/Autodownload.cmake

```diff
@@ -45,9 +45,11 @@ macro(get_deps LINK DEPS_NAME PACKAGE)
   if (DIRECTORIES_LEN GREATER 0)
     list(GET DIRECTORIES 0 DEPENDENCY_DIR)
     set(GENERIC_INCLUDE_DIR "${CMAKE_BINARY_DIR}/deps/${DEPENDENCY_DIR}/include")
-    # Clean this line when boost is removed.
+    install(DIRECTORY "${GENERIC_INCLUDE_DIR}" DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}")
```

Comment:
I think you need to ensure that `DEPS_NAME` isn't `"boost"` for this line; otherwise we end up with some error like this:

```
CMake Error at cmake_install.cmake:57 (file):
  file INSTALL cannot find "/root/mlpack/build/deps/boost_1_76_0/include": No
  such file or directory.
```

Reply:
Exactly, I will do the modification related to this one